jujugui/static/gui/src/app/app.js

```diff
@@ -920,7 +920,8 @@ YUI.add('juju-gui', function(Y) {
         // Switch to the redirected model.
         this.switchEnv(this.createSocketURL(
           this.get('socketTemplate'),
-          this.get('modelUUID'), publicHost.value, publicHost.port));
+          this.get('modelUUID'), publicHost.value, publicHost.port), null,
+          null, null, true, false);
```

Comment:
Would it be worth adding a comment here to explain the state these nulls and bools are setting?